assets/js/components/landing_page/overrides/RecordManagement.js

```diff
@@ -0,0 +1,140 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2023 CERN.
+//
+// Zenodo RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import React, { useEffect, useState } from "react";
+import { Button, Grid, Message, Icon } from "semantic-ui-react";
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import { EditButton } from "@js/invenio_app_rdm/landing_page/EditButton";
+import { ShareButton } from "@js/invenio_app_rdm/landing_page/ShareButton";
+import { NewVersionButton } from "@js/invenio_rdm_records/";
+import PropTypes from "prop-types";
+import { http, withCancel } from "react-invenio-forms";
+
+export const ZenodoRecordManagement = ({
+  permissions,
+  isDraft,
+  recid,
+  handleError,
+  isPreviewSubmissionRequest,
+  record,
+  currentUserId,
+  error,
+}) => {
+  const [requests, setRequests] = useState(null);
+
+  useEffect(() => {
+    const fetchRequests = async () => {
+      const results = withCancel(
+        http.get(`/api/records/${record.id}/requests`, {
+          headers: {
+            Accept: "application/json",
+          },
+        })
+      );
+
+      try {
+        const response = await results.promise;
+        setRequests(response.data.hits.hits);
+      } catch (error) {
+        console.error(error);
+      }
+    };
+
+    if (record.parent.access.owned_by[0].user == currentUserId) {
+      fetchRequests();
+    }
+  }, []);
+
+  const isLegacyRecord = () => {
```

Comment:
you can filter by type straight away in the endpoint, no need to filter it in the frontend